hnet/handler.go

```diff
@@ -64,6 +64,14 @@ func handleStatusChange(stream *common.IOStream, player *Player) error {
 	}
 
 	player.Status = status
+
+	statsResponse := StatsResponse{
+		Stats:  player.Stats,
+		Status: player.Status,
+	}
+
+	player.Server.Players.Broadcast(SERVER_USER_STATS, statsResponse)
```

Comment:
Please don't broadcast the status change. The clients will request the stats themselves every few seconds, when they have the player list open.